### PR TITLE
Fix failing lustre tests

### LINKED ISSUE #210: Failing lustre tests
Running pytest on branch v090dev with lustre mounted fails on these tests:

```
FAILED pyaerocom/io/test/test_read_aeronet_sdav3.py::test_load_berlin_AeroSdaV2L2D - OSError: No files could be detected that match the pattern
FAILED pyaerocom/io/test/test_read_aeronet_sunv3.py::test_load_berlin - OSError: No files could be detected that match the pattern
FAILED pyaerocom/io/test/test_read_ghost.py::TestReadGhost::test__eval_flags_slice - PermissionError: [Errno 13] Permission denied: b'/lustre/storeA/project/aerocom/aerocom1/AEROCOM_OBSDATA/GHOST/data/EEA_AQ_...
FAILED pyaerocom/io/test/test_read_ghost.py::TestReadGhost::test_read_file[ghost_eea_daily-2290-Bleak House] - PermissionError: [Errno 13] Permission denied: b'/lustre/storeA/project/aerocom/aerocom1/AEROCOM_...
FAILED pyaerocom/io/test/test_readgridded.py::test_years_available - AssertionError: 
FAILED pyaerocom/io/test/test_readungridded.py::test_basic_attributes - AssertionError: assert ['od340aer', ...4&87aer', ...] == ['od340aer', ...er_calc', ...]
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_ungriddeddata_jungfraujoch - assert (17520, 12) == (245448, 12)
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_scat_jungfraujoch - assert 8323 == 25789
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_scat_jungfraujoch_lev3 - assert 8323 == 25789
FAILED pyaerocom/test/test_ebas_sample_stats.py::test_scat_jungfraujoch_subset - pyaerocom.exceptions.DataCoverageError: ['sc550aer'] data could not be retrieved for meta  index (or station name) [0.0, 1.0]
FAILED pyaerocom/test/test_obsdata_versions.py::test_revision_dates - AssertionError: 
```

The two tests related to read_ghost is caused by the new ghost data having more restrictive permissions than other aerocom data. @hansbrenna can you update that? Other files have 664 permissions so they are readable but not writable even if not in the aerocom group.

For the others I'm not sure what is going on. Has the datasets been changed recently?

### PR DESCRIPTION
**Pull request by ejgal**
Closes #210

### CODE REVIEW

**Comment by ejgal** [comment]
@jgliss, can you have a quick look at this?

File: pyaerocom/io/test/test_read_aeronet_sdav3.py [hunk 1]
```diff
@@ -10,12 +10,11 @@
 from pyaerocom.conftest import TEST_RTOL, lustre_unavail
```
Comment (ejgal) [inline pyaerocom/io/test/test_read_aeronet_sdav3.py:10]:
```suggestion
from pyaerocom.conftest import TEST_RTOL, testdata_unavail
```

File: pyaerocom/io/test/test_read_aeronet_sdav3.py [hunk 2]
```diff
@@ -10,12 +10,11 @@
...
 from pyaerocom.io.read_aeronet_sdav3 import ReadAeronetSdaV3
```
Comment (ejgal) [inline pyaerocom/io/test/test_read_aeronet_sdav3.py:12]:
```suggestion

@testdata_unavail
```

**Review by jgliss — APPROVED** [review]
All good, thanks for fixing this!

**Pull request merged**